core/server/worker/src/test/java/alluxio/worker/block/NoopBlockWorker.java

```diff
@@ -48,6 +48,11 @@
     return null;
   }
 
+  @Override
+  public AtomicReference<String> getClusterId() {
+    return null;
```

Comment:
It doesn't matter very much here in the test, but generally speaking returning null is not the rule of thumb. Instead return something like `new AtomicReference("");`